Moku/Helpers/MapHelper.swift

```diff
@@ -89,6 +88,33 @@ struct MapHelper {
 
         return "N/A"
     }
+
+    static func direct(bengkel: Bengkel) {
+        let gmapAvailable = UIApplication.shared.canOpenURL(URL(string: "comgooglemaps://")!)
```

Comment:
Put all the function contains ".shared" in available Os mode, since this will no longer used in 15.

Reply:
@zeinrezky ada artikel for related matters ga kak? drtd kyknya nemu yang deprecated di iOS 15 cuma `UIApplication.shared.windows` kak